packages/utils/src/normalize.ts

```diff
@@ -224,6 +225,10 @@ function stringifyValue(
       return '[NaN]';
     }
 
+    if (typeof value === 'number' && !Number.isFinite(value)) {
```

Comment:
actually, we can improve this even further :D we can combine this with the check above for nan, as `(NaN).toString()` also returns `NaN`. And `isFinite(NaN) === false`. So we can remove this completely:

```js
if (typeof value === 'number' && value !== value) {
      return '[NaN]';
    }
```

and only keep the new check you added!